stanford_profile.post_update.php

```diff
@@ -104,9 +104,24 @@ function stanford_profile_post_update_8013() {
   /** @var \Drupal\stanford_notifications\NotificationServiceInterface $notifications */
   $notifications = \Drupal::service('notification_service');
 
-  $message = 'You can now create "News" content. See <a href="https://userguide.sites.stanford.edu/tour/news">the user guide</a> for more information';
-  $notifications->addNotification($message, ['site_manager', 'contributor']);
+  $message = 'New: You can now create "News" content. See <a href="https://userguide.sites.stanford.edu/tour/news">the user guide</a> for more information';
+  $notifications->addNotification($message, [
+    'site_manager',
+    'site_editor',
+    'contributor',
+  ]);
 
-  $message = 'You can now create "Person" content. See <a href="https://userguide.sites.stanford.edu/tour/person">the user guide</a> for more information';
-  $notifications->addNotification($message, ['site_manager', 'contributor']);
+  $message = 'New: You can now create "Person" content. See <a href="https://userguide.sites.stanford.edu/tour/person">the user guide</a> for more information';
+  $notifications->addNotification($message, [
+    'site_manager',
+    'site_editor',
+    'contributor',
+  ]);
+
+  $message = 'Update: All Paragraphs within a page will now support a maximum of 3 items per row where some allowed 4 items before. Any existing layouts with 4 items in a row will be grandfathered in but new content will be limited to 3 items per row.';
+  $notifications->addNotification($message, [
```

Comment:
I think to avoid confusion to users, we should only send the notification to those who have used 4. heres the query: `$query = 'select * from node_revision__su_page_components where su_page_components_settings like \'%"index":3%\'';`

Reply:
Thanks for the query. Have another go.